src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -112,6 +112,14 @@ private CallbackValidations() {
             : invalid("Exception case has no Id");
     }
 
+    @Nonnull
+    public static Validation<String, CaseDetails> hasId(CaseDetails theCase) {
+        return theCase != null
+            && theCase.getId() != null
+            ? valid(theCase)
+            : invalid("Exception case has no Id");
+    }
```

Comment:
all other validation methods in this class follow convention `hasX` returns `X`.
There's already a method that validates whether the case has Id.

Can you just call it and use `CaseDetails` object that you pass to that method afterwards?
The validation method does not change it so you can use it.